src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    private String msg;
+    private Boolean status;
```

Comment:
Since this is a Boolean, perhaps give it a more boolean name like isStatus as recommended.

Reply:
thanks, resolved